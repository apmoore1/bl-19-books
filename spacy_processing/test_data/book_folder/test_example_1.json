[[1, "Test OCR quality with sooooooommmmmmeeeee awwwwwfffful spelling."]]