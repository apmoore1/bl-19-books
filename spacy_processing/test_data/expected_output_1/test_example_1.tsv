token	pos	ner	page
<quality value="0.7143" />
<token count="8" />
<book identifier="test" />
Test	NN		1
OCR	NNP		1
quality	NN		1
with	IN		1
sooooooommmmmmeeeee	NN		1
awwwwwfffful	JJ		1
spelling	NN		1
.	.		1
