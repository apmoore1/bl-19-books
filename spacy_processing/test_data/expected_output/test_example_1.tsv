token	lemma	pos	ner	page
<quality value="0.7143" />
<token count="8" />
<book identifier="test" />
Test	test	NN		1
OCR	OCR	NNP		1
quality	quality	NN		1
with	with	IN		1
sooooooommmmmmeeeee	sooooooommmmmmeeeee	NN		1
awwwwwfffful	awwwwwfffful	JJ		1
spelling	spelling	NN		1
.	.	.		1
